content/docs/deployment/csminstallationwizard/_release_notes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Release Notes
+description: Release notes for CSM Installation Wizard
+---
+
+## Release Notes - CSM Installation Wizard 1.1.0
+
+### New Features/Changes
+
+- Added operator mode of installation for CSI-PowerStore, CSI-PowerMax, CSI-PowerScale and the supported modules
+
+- Helm and Operator based manifest file generation is supported for CSM-1.7 and CSM 1.8 releases
+
+### Fixed Issues
+
+- [#959 - [BUG]: Tolerations in the generated values.yaml should be uncommented when resiliency is enabled](https://github.com/dell/csm/issues/959)
```

Comment:
Tolerations?
Should this be Tolerances?

Reply:
Addressed